### DISCUSSION TITLE
Perfil com foto precisa ser removido.

### DISCUSSION THREAD

**Discussion by jonas-andrade** [community/community#38612] [Other Feature Feedback, Questions, & Ideas]
Oi por favor meu nome é Jonas Medeiros Andrade. Eu possuo um perfil a qual eu não tenho como recuperar a senha gostaria muito de deletar pois contém minha foto...
Eu sou o dono da conta mas não tenho nenhum acesso. ...a conta que precisa ser removido é github.com/jonas-medeiros

**Comment by radar-blip** [comment]
Only GitHub Support would  be able to help you with that, here is a link: [Support Page](https://support.github.com/request)
